client/my-sites/checkout/composite-checkout/lib/get-plan-features.ts

```diff
@@ -67,11 +68,10 @@ export default function getPlanFeatures(
 	if ( isWpComBusinessPlan( productSlug ) || isWpComProPlan( productSlug ) ) {
 		return [
 			! isMonthlyPlan && freeOneYearDomain,
-			! isMonthlyPlan && liveChatSupport,
-			String( translate( 'Install custom plugins and themes' ) ),
-			String( translate( 'Drive traffic to your site with our advanced SEO tools' ) ),
-			String( translate( 'Track your stats with Google Analytics' ) ),
-			String( translate( 'Real-time backups and activity logs' ) ),
+			String( translate( 'Install Custom Plugins and themes' ) ),
+			String( translate( 'Sell products with WooCommerce' ) ),
+			String( translate( '50GB of Storage' ) ),
+			String( translate( 'Free professional e-mail for 3 months' ) ),
```

Comment:
I would have expected "professional e-mail" to read as "Professional Email" because the latter is the [name of the product](https://wordpress.com/professional-email/). My personal preference would be to not even list the product name, because it doesn't explain much to a new user. I'd rather we said something on the lines "Free personal email inbox for 3 months" or something on those lines but I defer to the copy editors.